esmvaltool/_data_finder.py

```diff
@@ -137,22 +153,25 @@ def get_input_filename(variable, rootpath, drs):
 
     This function should match the function get_input_filelist below.
     """
-    dirname_template = get_input_dirname_template(variable, rootpath, drs)
-    # Simulate a latest version if required
-    if '[latestversion]' in dirname_template:
-        part1, part2 = dirname_template.split('[latestversion]')
-        dirname = os.path.join(part1, 'latestversion', part2)
-    else:
-        dirname = dirname_template
+    all_files = []
```

Comment:
This function should return a single filename, just use the first item of `dirname_templates` and leave the rest unchanged.